jaeger-mixin/alerts.libsonnet

```diff
@@ -14,27 +14,27 @@ local percentErrsWithTotal(metric_errs, metric_total) = '100 * sum(rate(%(metric
       {
         name: 'jaeger_alerts',
         rules: [{
-          alert: 'JaegerHTTPServerErrs',
-          expr: percentErrsWithTotal('jaeger_agent_http_server_errors_total', 'jaeger_agent_http_server_total') + '> 1',
+          alert: 'JaegerAgentUDPPacketsBeingDropped',
+          expr: 'rate(jaeger_agent_thrift_udp_server_packets_dropped_total[1m]) > 1',
```

Comment:
Can this be done as a % of total packed?  I don't like absolute thresholds.

Reply:
The main reason I did it this way because with UDP server dropping, it's because the UDP queue is full, and just checking for dropping makes more sense than a percentage here.

I'd ideally like to know packets are being dropped vs if only `>1%` of the packets are being dropped no? Atleast for the case of UDP?